test/nodes/test_tool_http_request.py

```
@@ -0,0 +1,453 @@
+# ---------------------------------------------------------------------------
+
+_SDK_MODULES = ['nodes.tool_http_request.http_client']
+_saved_sdk_modules = {name: sys.modules[name] for name in _SDK_MODULES if name in sys.modules}
+
+_mock_http_client = types.ModuleType('nodes.tool_http_request.http_client')
+_mock_http_client.execute_request = MagicMock(return_value={'status': 200, 'body': 'ok'})
+sys.modules['nodes.tool_http_request.http_client'] = _mock_http_client
+
+
+@pytest.fixture(autouse=True, scope='module')
+def _restore_http_sdk_modules():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_restore_http_sdk_modules`

(ANN202)